test/handlebar-helpers.js

```diff
@@ -92,4 +92,16 @@ describe('handlebar-helper', function () {
         ret = HandlebarHelper.assign(data, 'what', options)
         assert.equal(options.data.root[data], 'what', "assign works correctly")
     });
+    it('should correctly divide 30 / 6', function () {
+        ret = HandlebarHelper.divide(30, 6)
+        assert.equal(5, ret, '30 divided by 6 and got 5');
+    });
+    it('should correctly divide 0 / 6', function () {
+        ret = HandlebarHelper.divide(0, 6)
+        assert.equal(0, ret, '0 can not divide by 6 and got 0');
+    });
+    it('should not correctly divide 30 / 0', function () {
+        ret = HandlebarHelper.divide(30, 0)
+        assert.equal('can not divide as v2 is 0', ret, '30 can not be divide by 0');
+    });
```

Comment:
please add more test cases for less common error/success paths:
1. 10/3
2. passing a non number value to helper

Reply:
@alewitt2 thanks for the inputs ... i added more UT cases

Reply:
also Travis build is failing at very initial point `The command "echo "$DOCKERHUB_TOKEN" | docker login -u "icdevops" --password-stdin" failed and exited with 1 during.`

Reply:
thanks. Ah yeah, forgot about that. i'll try to fix that quickly